index.html

```diff
@@ -21,24 +21,24 @@ <h1 class="mb-0 bold">Web Accessor-Ease</h1>
   <main>
     <!-- Color Contrast Input -->
     <section id="colors" class="panel-section mb-3 p-3">
-      <h2>Colour Contrast</h2>
+      <h2>Clarity & Colour</h2>
       <div>
         <input type="checkbox" id="color-contrast" data-key="colorContrast"
           aria-label="Toggle to fix any colors with low contrast" checked />
-        <label for="color-contrast">Fix Low Colour Contrast</label>
+        <label for="color-contrast">Increase Colour Contrast</label>
```

Comment:
I think my description is more fitting. It doesn't increase the colour contrast for everything. It just finds text with low colour contrast and adjusts it